android/app/src/main/java/app/priceguard/ui/login/LoginViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package app.priceguard.ui.login
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+class LoginViewModel : ViewModel() {
+    data class State(
+        val email: String = "",
+        val password: String = ""
+    )
+
+    private val _state = MutableStateFlow(State())
+    var state: StateFlow<State> = _state.asStateFlow()
+
+    fun setID(s: CharSequence, start: Int, before: Int, count: Int) {
+        _state.value = _state.value.copy(email = s.toString())
+    }
+
+    fun setPassword(s: CharSequence, start: Int, before: Int, count: Int) {
+        _state.value = _state.value.copy(password = s.toString())
+    }
+
+    fun logIn(onSuccess: (Unit) -> Unit) {
```

Comment:
사소한거긴 한데 로그인은 한단어라 lowerCamelCase 처리가 필요없을 듯 합니다.
```suggestion
    fun login(onSuccess: (Unit) -> Unit) {
```

Reply:
그리고 제가 [이 부분](https://github.com/boostcampwm2023/kotlin-p4-onban/pull/70#discussion_r1365309190)에 대해서 리뷰어분께 지적받은 적이 있는데 같이 논의해보면 좋을 것 같습니다.